lib/tasks/data_fixes.rake

```diff
@@ -17,4 +18,17 @@ namespace :data_fixes do
          "patients: #{patient_count}, BPs: #{bp_count}, blood sugars: #{bs_count}, "\
          "appointments: #{appointment_count}, prescriptions: #{prescription_drug_count}"
   end
+
+  desc "Clean up invalid scheduled appointments (multiple scheduled appointments for a patient)"
+  task discard_invalid_scheduled_appointments: :environment do
+    patients_ids = Appointment
```

Comment:
Should we pass all patients to `DiscardInvalidAppointments` and do this there? This logic and the looping could go in there then.

Reply:
Thought about it, but decided to go this way to be a little more consistent. 
The other `data_fixes` script here also operates on one (user) record at a time.